### PR TITLE
001 delivery emp

### PR DESCRIPTION
**Pull request by RaggedyRach**
please review our PR for creating employee (delivery) front end

### CODE REVIEW

File: src/models/employeeResponse.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+export type employeeResponse = {
```
Comment (shaunganley) [inline src/models/employeeResponse.ts:1]: is this the same as employee.ts?

File: views/employeeForm.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                <label for="commRate">
+                    Commission Rate: 
+                </label></h3>
+            <input id="commRate" name="commRate" value="{{ commRate }}" type="number">
+
+            <h3>
+                <label for="roleID">
+                   Delivery Employee: 
+                </label></h3>
+            <input id="roleID" name="roleID" value="2" type="number">
```
Comment (shaunganley) [inline views/employeeForm.html:52]: this isn't needed?

**Review by shaunganley — APPROVED** [review]
I have approved but the requirement is the ability to create delivery employee and all of the endpoints are for generic employees

File: src/models/employeeResponse.ts (hunk 1 above)
Comment (RaggedyRach) [reply src/models/employeeResponse.ts:1]: deleted employee.ts

**Pull request merged**